src/css/utils/flex.scss

```diff
@@ -73,6 +73,10 @@
   justify-content: space-around !important;
 }
 
+.u-flexJustifyEvenly {
+  justify-content: space-evenly !important;
```

Comment:
I don't know how, but I completely missed the existence of this flex property. It's weird that SUIT CSS doesn't include it either 🤷‍♀️

Reply:
No worries :) I tried a few examples of what I thought it would be and when none of them worked, I realized we were missing it ! D:

Reply:
Nice, TIL!